### ISSUE TITLE
scale functions ?

### ISSUE THREAD

**Issue by postite** [fponticelli/thx.math#5]
hello @fponticelli ..
are you planning to add scaling function from the old thx repo  to thx.math ?https://github.com/fponticelli/thx/tree/master/src/thx/math/scale
or will it be in a separate project ?
thx is so helpfull  , i'd love to move to those next versions but i still need scaling.

thx for thx

**Comment by fponticelli** [comment]
Yes, I am planning to do it but I have not decided yet on the ultimate format :)
